tests/test_storage.py

```diff
@@ -1,9 +1,31 @@
 import pytest
 
+from peerdid.peer_did import save_peer_did
+from peerdid.peer_did_utils import encode_filename
+from peerdid.storage import FileStorage
 
-def test_save():
-    pass
+
+def test_save_positive():
+    peer_did = 'did:peer:0z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V'
+    file_storage = FileStorage(peer_did_filename=encode_filename(peer_did))
```

Comment:
For future: The tests should not write to non-tmp locations during execution. 
Consider using `with tempfile.TemporaryDirectory() as temp_dir:` and pass the temp_dir as a root dir.